pkg/sidecar/k8s_reactor.go

```diff
@@ -37,17 +37,26 @@ var (
 )
 
 type K8sReactor struct {
-	client  *sync.Client
-	redis   net.IP
-	manager *docker.Manager
+	client         *sync.Client
+	manager        *docker.Manager
+	servicesRoutes []net.IP
 }
 
 func NewK8sReactor() (Reactor, error) {
-	redisHost := os.Getenv(EnvRedisHost)
+	// TODO: Generalize this to a list of services.
+	wantedRoutes := []string{
+		os.Getenv(EnvRedisHost),
+		os.Getenv(EnvInfluxdbHost),
```

Comment:
Since this logic is getting more sophisticated, and we will likely want to debug sidecar sooner or later, it would be good to make this a `map[string]string` of `service name => IP/hostname`, so that we can print out the service name in log messages below.

Reply:
Does it makes sense to have a single env variable? I'm thinking like a comma or colon-delimeted string 

```
SIDECAR_ALLOW=service1:service2:service3
```

Reply:
We should totally add a mapping from service name to service that we want to allow in the sidecar. I will see about address this in a future PR.